CheckYourEligibility-Admin/Views/Check/ApplicationsRegistered.cshtml

```diff
@@ -32,25 +37,23 @@
                 @foreach (var item in @Model.Children)
                 {
                     <tr class="govuk-table__row">
-                        <td class="govuk-table__cell">@item.ParentName</td>
                         <td class="govuk-table__cell">@item.ChildName</td>
                         <td class="govuk-table__cell">@item.Reference</td>
                     </tr>
                 }
-
             </tbody>
         </table>
     </div>
 </div>
+
 <div class="govuk-width-container">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
             <h2 class="govuk-heading-m">What to do next</h2>
             <p> @TempData["Status"]</p>
```

Comment:
This is where the if clause could be. I believe we know what it is without having to use tempdata, just look at the status of the returned freshly minted application